### DISCUSSION TITLE
azure-workload-identity `v1.0.0` released  🎉

### DISCUSSION THREAD

**Discussion by aramase** [Azure/azure-workload-identity#813] [Announcements]
https://github.com/Azure/azure-workload-identity/releases/tag/v1.0.0

**Comment by brandongodby78** [comment]
@aramase Hi Anish, does this mean that official GA for workload identity is immanent? Very excited to deploy this feature.

**Reply by brandongodby78** [reply]
Forgive me if this is the announcement of GA :-)

**Reply by aramase** [reply]
This is the stable release for the OSS project. If you're using the AKS add-on, the GA announcement will be before end of March.

**Reply by aramase** [reply]
AKS GA announcement: https://techcommunity.microsoft.com/t5/apps-on-azure-blog/general-availability-for-azure-active-directory-ad-workload/ba-p/3798292

**Comment by mo-saeed** [comment]
Hi @aramase 

did I understand the breaking change correctly, that from version 1, the label `"azure.workload.identity/use"` must be now on the pod level not on the service account?

**Reply by aramase** [reply]
That's right! We made the change in backward compatible way in `v0.15.0`. With `v1.0.0` only if the pod is labelled, the pod will be mutated by the mutating webhook. The label in service account is no longer required.

https://github.com/Azure/azure-workload-identity/blob/4ce591ec5f89954f78ae9fc6b7be0a0f5390773f/deploy/azure-wi-webhook.yaml#L254-L256
